proto/message.py

```diff
@@ -558,7 +558,7 @@ def to_dict(
         including_default_value_fields=None,
         float_precision=None,
         always_print_fields_with_no_presence=None,
-    ) -> "Message":
+    ) -> dict[str, Any]:
```

Comment:
Using `Dict` will unblock this PR as the repository still has support for Python 3.8

Reply:
done! Thanks for the guidance.